airflow_versions/http_client.go

```diff
@@ -0,0 +1,70 @@
+package airflowversions
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"strings"
+
+	"github.com/astronomer/astro-cli/config"
+	"github.com/astronomer/astro-cli/pkg/httputil"
+	"github.com/pkg/errors"
+)
+
+// Client containers the logger and HTTPClient used to communicate with the HoustonAPI
+type Client struct {
+	HTTPClient *httputil.HTTPClient
+}
+
+// NewClient returns a new Client with the logger and HTTP client setup.
+func NewClient(c *httputil.HTTPClient) *Client {
+	return &Client{
+		HTTPClient: c,
+	}
+}
+
+type Request struct{}
+
+// Do (request) is a wrapper to more easily pass variables to a client.Do request
+func (r *Request) DoWithClient(api *Client) (*Response, error) {
+	doOpts := httputil.DoOptions{
+		Headers: map[string]string{
+			"Accept": "application/json",
+		},
+	}
+
+	return api.Do(doOpts)
+}
```

Comment:
Will this function be updated? It does not look like you are using the `r` struct here and you may have a duplicate comment (line 28 and 39)

Reply:
@adam2k @Simpcyclassy fixed PTAL!.